conf/application.conf

```diff
@@ -135,6 +133,7 @@ metrics {
     enabled = true
 }
 
+play.i18n.langs = [ "en-GB", "en", "en-US"]
```

Comment:
is this being used anywhere? If anything we should probably only support `en-GB`

Reply:
could we make this "en" like the FE?